### PR TITLE
Fleshshot Repair

### PR DESCRIPTION
**Pull request by Shadowmech88**
I've gone through the fleshshot's code and taken out all of the copy-paste. I also added a fire delay to the fleshshot item.
The clicks that I get when I attempt to fire it while extended seem as slow as they are on the hookshot, and I no longer get the weird icon behavior when spamming it, so I *think* the crashing issue has been fixed. I can't personally test that as I never got it to crash reliably, but having the PR made will let me load it on the PR test server so others can test it.

:cl:
 * rscadd: The fleshshot has been repaired and re-added to borers, now verified crash-free.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
@Unknown-as-Captain 
You've been able to reproduce the crashing consistently, so come into the coder Discord when you can and ping me, and I'll load this up on the PR test server for you.

File: code/modules/projectiles/projectile/hookshot.dm [hunk 1]
```diff
@@ -8,6 +8,10 @@
 	kill_count = 15
 	grillepasschance = 0
 	var/obj/effect/overlay/hookchain/last_link = null
+	var/failure_message = "With a CLANG noise, the chain mysteriously snaps and rewinds back into the hookshot."
```
Comment (N3X15) [inline code/modules/projectiles/projectile/hookshot.dm:11]: > CLANG
crash course on clang

DO NOT INVOKE HIS NAME

Comment (Zth--) [reply code/modules/projectiles/projectile/hookshot.dm:11]: _With a CLANG noise, the server mysteriously snaps and rewinds back into the void_

**Comment by ghost** [comment]
No crashes or graphical issues anymore, fleshshot produces clickclick sound though.

**Comment by Shadowmech88** [comment]
I had it producing the clicking sound for testing purposes, I'll remove that.

**Comment by Shadowmech88** [comment]
Oh, also with the code as it is, the item overlay on the end of the fleshshot fails to appear when the fleshshot is retracting, but only when fired at certain angles. It appears for 90% of the angles.
I can't figure out why, but it's nowhere near as urgent a problem as the crashing is.

**Comment by Shadowmech88** [comment]
This is ready to go now.

**Pull request merged**